src/chat.rs

```diff
@@ -663,7 +663,7 @@ fn prepare_msg_common(context: &Context, chat_id: u32, msg: &mut Message) -> Res
             msg.type_0
         );
 
-        let mut path_filename = path_filename.unwrap().to_string();
+        let mut path_filename = path_filename.unwrap_or_default().to_string();
```

Comment:
the previous statement does and ensure path_filename.is_some().  this unwrap can't fail